modules/tensor_mechanics/test/tests/2D_different_planes/tests

```diff
@@ -0,0 +1,185 @@
+[Tests]
+  [./plane_strain_xy_small]
+    type = Exodiff
+    input = 'planestrain_xy.i'
+    exodiff = 'planestrain_xy_small_out.e'
+    abs_zero = 1e-7
```

Comment:
Time to start adding in the new "requirement", "design", and "issues" tags for NQA-1. There's a lot of new tests here...

Reply:
The design parameter pointing to the .md file for documentation for some of these tests is included in for the plane strain documentation included in merge request: #11614 and can't be added yet. All the other tests I can add.